packages/react/README.md

```diff
@@ -763,6 +764,48 @@ export function DummyProducts(props) {
 
 `useFetch` will **not** send your Gadget API client's authentication headers to third party APIs. It will behave like a normal browser `fetch` call, just with the added React wrapper and `json: true` option for easy JSON parsing.
 
+#### When the request gets sent
+
+By default, `useFetch` will immediately issue HTTP requests for `GET`s when run. This makes it easy to use `useFetch` to retrieve data for use rendering your component right away.
+
+```javascript
+export function GetRequest(props) {
+  // will automatically send the request when the component renders the first time, as it is a GET
+  const [{ data, fetching, error }, resend] = useFetch("/products");
+  // fetching will be `true`
+}
+```
+
+`useFetch` will _not_ immediately issue HTTP requests for HTTP verbs other than `GET`, like `POST`, `PUT`, etc. The HTTP request will only be sent when you call the returned `send` function.
+
+```javascript
+export function PostRequest(props) {
+  // will not automatically send the request when the component renders -- call the `send` function to run the request
```

Comment:
super nit: "call the `send` function to run the request" is a bit redundant with the comment below about fetching being false. Either way, maybe we have the comments for this example and the delayed get below be exactly the same?